SNS MP2 installation fails

I'm compiling the latest master of Psi4. SNS-MP2 automatic download failed with 
``Cloning into 'snsmp2_external'...
fatal: invalid reference: a035c27
CMake Error at /home/jerome/Documents/Atlanta/PSIcode/psi4public_JFG/obj_rel/external/downstream/snsmp2/snsmp2_external-prefix/tmp/snsmp2_external-gitclone.cmake:75 (message):
  Failed to checkout tag: 'a035c27'
``
Looking up the SNS-MP2 Git I found the tag of the current release to be dd286d3. I replaced it in ``external/downstream/snsmp2/CMakeLists.txt`` and the compilation proceeded.
Is this actually the correct version for SNS-MP2? Should I submit a PR with this tiny change?

--- a/external/downstream/snsmp2/CMakeLists.txt
+++ b/external/downstream/snsmp2/CMakeLists.txt
@@ -19,10 +19,8 @@ if(${ENABLE_snsmp2})
         ExternalProject_Add(snsmp2_external
             DEPENDS qcelemental_external
             BUILD_ALWAYS 1
-            GIT_REPOSITORY https://github.com/loriab/sns-mp2
-            GIT_TAG a035c27
-            #GIT_REPOSITORY https://github.com/DEShawResearch/sns-mp2
-            #GIT_TAG a70c56f  # v1.0 + 13
+            GIT_REPOSITORY https://github.com/DEShawResearch/sns-mp2
+            GIT_TAG 99e2a9c  # v1.0 + 16
             CONFIGURE_COMMAND ""
             UPDATE_COMMAND ""
             BUILD_COMMAND ${PYTHON_EXECUTABLE} setup.py build

